apps/web/src/pages/Pools/ViewPoolPage/components/DuplicateProcessDialog.tsx

```diff
@@ -53,43 +100,81 @@ const DuplicateProcessDialog = ({
               },
             )}
           </p>
-          <p data-h2-margin="base(x1, 0)">
-            {intl.formatMessage({
-              id: "NivtzV",
-              defaultMessage:
-                "This will create a new process and copy all existing information.",
-              description:
-                "Text explaining what will happen when duplicating a process",
-            })}
-          </p>
-          <p data-h2-margin="base(x1, 0)">
-            {intl.formatMessage({
-              id: "3Rad8l",
-              defaultMessage:
-                "Do you wish to continue? This will navigate away from this page.",
-              description:
-                "Text explaining what will happen after duplicating a pool and confirming the action",
-            })}
-          </p>
+          <FormProvider {...methods}>
+            <form onSubmit={handleSubmit(handleDuplicate)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  id: "9XaTwh",
+                  defaultMessage:
+                    "Specify the following information for this new recruitment:",
```

Comment:
Something small, but we could use the `commonMessages.dividingColon` here instead of including it in the string itself.

Reply:
oh, good idea
[use dividing colon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10698/commits/1214ea67add81686d41287e31bf976aa64c6d7da)